cdap-docs/_common/common-build.sh

```diff
@@ -355,7 +360,16 @@ function display_version() {
   echo "GIT_BRANCH: ${GIT_BRANCH}"
   echo "GIT_BRANCH_TYPE: ${GIT_BRANCH_TYPE}"
   echo "GIT_BRANCH_PARENT: ${GIT_BRANCH_PARENT}"
-  echo "GIT_BRANCH_CDAP_HYDRATOR: ${GIT_BRANCH_CDAP_HYDRATOR}"
+  echo "GIT_BRANCH_CASK_HYDRATOR: ${GIT_BRANCH_CASK_HYDRATOR}"
+  echo "CASK_HYDRATOR_VERSION: ${CASK_HYDRATOR_VERSION}"
+}
+
+function get_cask_hydrator_version() {
+  # $1 Branch of Hydrator Plugins to use
```

Comment:
The branch that is used — which is specified in the `vars` file as `GIT_BRANCH_CDAP_HYDRATOR` should be a release branch. That should *not* be a `SNAPSHOT` branch. Otherwise, how can you build a CDAP release against it? 

But it is whatever it is; you set the branch, and if Hydrator is using a `SNAPSHOT` branch, well, that needs to be fixed...